Last-Update: 2016-02-15
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: fix compilation without non-DFSG Selenium "ghostdriver/third_party/webdriver-atoms/*" files

--- a/src/ghostdriver/ghostdriver.qrc
+++ b/src/ghostdriver/ghostdriver.qrc
@@ -17,55 +17,8 @@
         <file>third_party/console++.js</file>
         <file>third_party/har.js</file>
         <file>third_party/parseuri.js</file>
         <file>third_party/uuid.js</file>
-        <file>third_party/webdriver-atoms/active_element.js</file>
-        <file>third_party/webdriver-atoms/clear.js</file>
-        <file>third_party/webdriver-atoms/clear_local_storage.js</file>
-        <file>third_party/webdriver-atoms/clear_session_storage.js</file>
-        <file>third_party/webdriver-atoms/click.js</file>
-        <file>third_party/webdriver-atoms/double_click.js</file>
-        <file>third_party/webdriver-atoms/drag.js</file>
-        <file>third_party/webdriver-atoms/execute_async_script.js</file>
-        <file>third_party/webdriver-atoms/execute_script.js</file>
-        <file>third_party/webdriver-atoms/execute_sql.js</file>
-        <file>third_party/webdriver-atoms/find_element.js</file>
-        <file>third_party/webdriver-atoms/find_elements.js</file>
-        <file>third_party/webdriver-atoms/focus_on_element.js</file>
-        <file>third_party/webdriver-atoms/get_appcache_status.js</file>
-        <file>third_party/webdriver-atoms/get_attribute.js</file>
-        <file>third_party/webdriver-atoms/get_attribute_value.js</file>
-        <file>third_party/webdriver-atoms/get_current_position.js</file>
-        <file>third_party/webdriver-atoms/get_local_storage_item.js</file>
-        <file>third_party/webdriver-atoms/get_local_storage_keys.js</file>
-        <file>third_party/webdriver-atoms/get_local_storage_size.js</file>
-        <file>third_party/webdriver-atoms/get_location.js</file>
-        <file>third_party/webdriver-atoms/get_location_in_view.js</file>
-        <file>third_party/webdriver-atoms/get_session_storage_item.js</file>
-        <file>third_party/webdriver-atoms/get_session_storage_keys.js</file>
-        <file>third_party/webdriver-atoms/get_session_storage_size.js</file>
-        <file>third_party/webdriver-atoms/get_size.js</file>
-        <file>third_party/webdriver-atoms/get_text.js</file>
-        <file>third_party/webdriver-atoms/get_value_of_css_property.js</file>
-        <file>third_party/webdriver-atoms/is_displayed.js</file>
-        <file>third_party/webdriver-atoms/is_enabled.js</file>
-        <file>third_party/webdriver-atoms/is_online.js</file>
-        <file>third_party/webdriver-atoms/is_selected.js</file>
-        <file>third_party/webdriver-atoms/lastupdate</file>
-        <file>third_party/webdriver-atoms/move_mouse.js</file>
-        <file>third_party/webdriver-atoms/pinch.js</file>
-        <file>third_party/webdriver-atoms/remove_local_storage_item.js</file>
-        <file>third_party/webdriver-atoms/remove_session_storage_item.js</file>
-        <file>third_party/webdriver-atoms/right_click.js</file>
-        <file>third_party/webdriver-atoms/rotate.js</file>
-        <file>third_party/webdriver-atoms/scroll_into_view.js</file>
-        <file>third_party/webdriver-atoms/scroll_mouse.js</file>
-        <file>third_party/webdriver-atoms/set_local_storage_item.js</file>
-        <file>third_party/webdriver-atoms/set_session_storage_item.js</file>
-        <file>third_party/webdriver-atoms/submit.js</file>
-        <file>third_party/webdriver-atoms/swipe.js</file>
-        <file>third_party/webdriver-atoms/tap.js</file>
-        <file>third_party/webdriver-atoms/type.js</file>
         <file>webdriver_atoms.js</file>
         <file>webelementlocator.js</file>
     </qresource>
 </RCC>
